src/main/groovy/org/transmartproject/batch/highdim/datastd/TripleDataValueWrappingReader.groovy

```diff
@@ -38,6 +38,9 @@ class TripleDataValueWrappingReader implements ItemReader<TripleStandardDataValu
     }
 
     private TripleStandardDataValue process(TripleStandardDataValue item) throws Exception {
+        if (item.value == null) {
+            return item
```

Comment:
This line is never hit in the tests. Why this null handling? I don't think you can get a `null` from the data file.